pyproject.toml

```diff
@@ -0,0 +1,61 @@
+#  - git pull --rebase
+#  - Remove untracked files/dirs: git clean -fdx
+#  - maybe update version in pyperformance/__init__.py and doc/conf.py
+#  - set release date in doc/changelog.rst
+#  - git commit -a -m "prepare release x.y"
+#  - run tests: tox --parallel auto
+#  - git push
+#  - check the CI status:
+#    https://github.com/python/pyperformance/actions
+#
+# Release a new version:
+#
+#  - git tag VERSION
+#  - git push --tags
+#  - Remove untracked files/dirs: git clean -fdx
+#  - python -m build
```

Comment:
This one line is the main difference in the workflow, right?

Where does the "build" module come from?  I don't have one installed.

Reply:
https://github.com/python/pyperformance/pull/249/commits/e5d73924218ffc32bf2ee35513f0e03d3f5d45df

Updated!